pytest_mh/utils/fs.py

```diff
@@ -611,3 +611,27 @@ def chmod(self, mode: str, path: str, args: list[str] = []) -> SSHProcessResult:
         self.logger.info(f'Changing mode bits for "{path}" on {self.host.hostname}')
 
         return self.host.ssh.exec(["chmod", *args, mode, path])
+
+    def chown(
+        self, path: str, user: str | None = None, group: str | None = None, args: list[str] = []
+    ) -> SSHProcessResult:
+        """
+        Change file owner and group.
+
+        :param path: Path to file
+        :type path: str
+        :param user: New file owner, if None then user remains same, defaults to None
+        :type user: str | None, optional
+        :param group: New file group, if None then group remains same, defaults to None
+        :type group: str | None, optional
+        :param args: Additional options, defaults to []
+        :type args: list[str], optional
+        :return: Result of process
+        :rtype: SSHProcessResult
+        """
+        self.backup(path)
+        self.logger.info(f'Changing owner for "{path}" on {self.host.hostname}')
```

Comment:
I have just merged https://github.com/next-actions/pytest-mh/pull/28 so you can avoid the hostname now. Please remove ` on {self.host.hostname}`. It would be nice to see the new owner in the debug message.

Reply:
Edited